Dockerfile

```diff
@@ -17,4 +17,6 @@ WORKDIR /tmp/connect
 RUN bundle -j8
 ADD . /tmp/connect
 RUN chown -R nobody /tmp/connect
-RUN chmod +x /tmp/connect/docker/runall.sh
+RUN chmod +x /tmp/connect/docker/prophet_tests.sh /tmp/connect/docker/integration.sh
+ADD .oscrc /root/.oscrc
+ADD .regcode /root/.regcode
```

Comment:
NB fetching them from gaffer into the image allows the image to be used standalone without knowing Jenkins config <- Tom